src/site/content/en/reliable/broadcast-updates-guide/index.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: post 
+title: "Broadcast Updates" 
+authors:
+  - demianrenzulli
+  - andrewguan 
+date: 2020-12-04 
+description: | 
+  How service workers can proactively communicate with the page to inform about certain events.
+tags:
+  - blog
+  - service-worker
+  - web-worker
+---
+
+In some scenarios the service worker might need to proactively communicate with any of the active
+tabs it controls to inform of a certain event. Examples include:
+
+- Informing the page when a new version of the service worker has been installed, so that the page
+  can show an **"Update to refresh"** button to the user to access the new functionality
+  immediately.
+- Letting the user know about a change on cached data that took place on the service worker side, by
+  showing an indication, like: **"The app is now ready to work offline"**, or **"New version of the
+  content available"**.
+
+<figure class="w-figure">
+  <img src="broadcast-updates-diagram.png"
+       width="550"
+       alt="Diagram showing a service worker communicating with the page to send an update.">
+</figure>
+
+We'll call these types of use cases where the service worker doesn't need to receive a message from
+the page to start a communication **“broadcast updates”**. In this article we'll review different
+ways of implementing this type of communication between pages and service workers, by using standard
+browser APIs and the [Workbox library](https://developers.google.com/web/tools/workbox).
+
+## Production cases
+
+### Tinder
+
+Tinder PWA uses [Workbox
+Window](https://developers.google.com/web/tools/workbox/modules/workbox-window) to listen to
```

Comment:
See imperative caching for reminder on how to format `workbox-window`, plz fix throughout the doc

Reply:
Done